mod/install.php

```diff
@@ -51,7 +51,7 @@ function install_post(App $a) {
 			$phpath = notags(trim($_POST['phpath']));
 
 			require_once("include/dba.php");
-			if (!dba::connect($dbhost, $dbuser, $dbpass, $dbdata)) {
+			if (!dba::connect($dbhost, $dbuser, $dbpass, $dbdata, true)) {
```

Comment:
You removed the fifth parameter of the `dba::connect` method, this shouldn't have been readded.